siddpubs-misc.bib

```diff
@@ -53,6 +53,14 @@ @inproceedings{nanavati2023unintended
   note = {PDF {\href{here}{https://personalrobotics.cs.washington.edu/publications/nanavati2023unintended.pdf}}}
 }
 
+@inproceedings{karim2023autonomy,
+  title = {Investigating the Levels of Autonomy for Personalization in Assistive Robotics},
+  author = {Karim, R. and Nanavati, A. and Faulkner, T.A.K. and Srinivasa, S. S.},
+  booktitle = hri,
+  year = {2023},
+  note = {PDF {\href{here}{https://personalrobotics.cs.washington.edu/publications/karim2023autonomy.pdf}}}
```

Comment:
Please remove this note. It was only necessary for the above publication since I wanted the main link to go to the YouTube video. In all other cases, the script will automatically like your title to the PDF, without needing a separate note.

Reply:
Done @amalnanavati !